src/nowplaying.html

```diff
@@ -31,49 +31,49 @@ <h2 class="nowPlayingPageTitle"></h2>
                     <div class="nowPlayingInfoButtons">
                     
                         <button is="paper-icon-button-light" class="btnRewind btnNowPlayingRewind btnPlayStateCommand autoSize" title="${Rewind}">
-                            <i class="material-icons replay_10"></i>
+                            <span class="material-icons replay_10"></span>
                         </button>
                         
                         <button is="paper-icon-button-light" class="btnPreviousTrack btnPlayStateCommand autoSize" title="${ButtonPreviousTrack}">
-                            <i class="material-icons skip_previous"></i>
+                            <span class="material-icons skip_previous"></span>
                         </button>
 
                         <button is="paper-icon-button-light" class="btnPlayPause btnPlayStateCommand autoSize" title="${ButtonPause}">
-                            <i class="material-icons">pause</i>
+                            <span class="material-icons pause_circle_filled"></span>
```

Comment:
why change icon?

Reply:
https://github.com/jellyfin/jellyfin-web/blob/d98645135b53612d852bf14991d247880f502265/src/components/remotecontrol/remotecontrol.js#L385

Earlier, it was replaced by `innerHTML = ` - it doesn't matter what icon is here. Now there is `replace`.

If we will do remove+add then we can leave any icon here.